gamla/functional_generic.py

```diff
@@ -133,7 +133,20 @@ def lazyjuxt(*args, **kwargs):
     return lazyjuxt
 
 
-juxt = compose(after(tuple), lazyjuxt)
+def juxt(*funcs):
+    if _any_is_async(funcs):
+        funcs = tuple(map(after(to_awaitable), funcs))
+
+        async def inner_juxt_async(*args, **kwargs):
```

Comment:
note that because you are changing the name, this may affect the names in `.verison` file in nlu-engine (it would raise an exception as if this kg is missing. To avoid that, you can let the kgs rebuild or just replace the lazyjuxt with juxt in the key names.

Reply:
how do i rebuild the kgs?